doc/man3/OPENSSL_malloc.pod

```diff
@@ -136,7 +136,9 @@ This adds some overhead by keeping a list of all memory allocations, and
 removes items from the list when they are free'd.
 This is most useful for identifying memory leaks.
 CRYPTO_set_mem_debug() turns this tracking on and off.  It is normally
-called at startup, but can be called at any time.
+called at startup, but in order to have any effect it must be called
+before any of the allocation functions (e.g., CRYPTO_malloc()) are done,
+including libraries internals.
```

Comment:
Yeah, "it is normally called at startup" could be (mis)read as "during normal operation, it is automatically called at startup".
It's probably better to start out with "In order to have any effect, it must be called before [...]" and maybe drop the "normally" entirely or end the sentence with ", so when used, it is usually called by the program during early program startup."